lang/en/auth_saml2.php

```diff
@@ -105,6 +105,8 @@
 $string['organizationname'] = 'Organisation';
 $string['passivemode'] = 'Passive mode';
 $string['pluginname'] = 'SAML2';
+$string['privatekeypass'] = 'Private certificate key';
```

Comment:
I think a slight tweak, this is not the cert key, it's the cert key password.

Reply:
yeah - good poiint - done.